### PR TITLE
css selectors

### PR DESCRIPTION
**Pull request by JulianO911**

### CODE REVIEW

File: cypress/e2e/buy-shirt.cy.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    checkout button */
+    cy.get(".cart_navigation button span").click();
+    /* click in the checkbutton wich has an id equals to 'cgv' */
+    cy.get("#cgv").click();
+    cy.get(".cart_navigation button span").click();
+    /* Click in the payment button which is in a bankwire class */
+    cy.get(".bankwire").click();
+    /* Click in the button to confirm the order wich has a span label and is
+    inside a label with an id equals to 'cart_navigation' */
+    cy.get("#cart_navigation button span").click();
```
Comment (kliver98) [inline cypress/e2e/buy-shirt.cy.ts]: Delete comments, I think it's not pretty helpful here. Will improve readability 
Something better you can do is group the steps into "section": example #email, #passwd and SubmitLogin are part of a page where user sign in. So all these three should be one line after the other, the last one is an space to differenciate another page.
```code
cy.stepA1
cy.stepA2
cy.stepA3

cy.stepB1

cy.stepC1
cy.stepC2
```

File: cypress.config.ts [hunk 2]
```diff
@@ -9,6 +9,8 @@ export default defineConfig({
     setupNodeEvents(on, config) {
       // modify config values examples
       // config.defaultCommandTimeout = 10000
```
Comment (kliver98) [inline cypress.config.ts]: delete these both please

File: cypress/e2e/buy-shirt.cy.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    ).click();
+    cy.get("[style*='display: block;'] .button-container > a").click();
+    cy.get(".cart_navigation span").click();
+
+    cy.get("#email").type("aperdomobo@gmail.com");
+    cy.get("#passwd").type("WorkshopProtractor");
+
+    /* click in a button that has id equals to 'SubmitLogin',
+    which is the Sign In button */
+    cy.get("button[id='SubmitLogin']").click();
```
Comment (kliver98) [inline cypress/e2e/buy-shirt.cy.ts]: Do it simpler, just use the identifier that is unique

File: cypress/e2e/buy-shirt.cy.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    /* Click in the span label which is inside a button label and, which at the
+    same time is inside in a class named cart_navigation. This is the 'proceed to
+    checkout button */
+    cy.get(".cart_navigation button span").click();
```
Comment (kliver98) [inline cypress/e2e/buy-shirt.cy.ts:22]: You could delete span that is specific. just click the button 
Same in line 23 and 28

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**